src/fidesops/db/session.py

```diff
@@ -27,7 +30,11 @@ def get_db_engine(database_uri: Optional[str] = None) -> Engine:
         else:
             database_uri = config.database.SQLALCHEMY_DATABASE_URI
 
-    return create_engine(database_uri, pool_pre_ping=True)
+    return create_engine(
+        database_uri,
+        pool_pre_ping=True,
+        isolation_level=isolation_level,
+    )
 
 
 def get_db_session(engine: Optional[Engine] = None) -> sessionmaker:
```

Comment:
Just setting `autocommit=True` here causes failures because we're calling `.commit()` in a lot of areas in the code, which fails if there isn't a transaction left open in the session.